content/articles/drop-catching.markdown

```diff
@@ -1,10 +1,26 @@
 ---
-title: Can I use DNSimple for Drop-catching?
-excerpt: Information about DNSimple drop-catching policy.
+title: Can I use DNSimple for Drop catching?
+excerpt: Information about DNSimple drop catching policy.
 categories:
 - DNSimple
+- Domain Registration
+- API
 ---
 
-# Can I use DNSimple for Drop-catching?
+# Can I use DNSimple for drop catching?
 
-No, DNSimple may not be used for drop-catching domains at this time.
+No, you may not use DNSimple for drop catching domains at this time.
+
+## What is drop catching?
+
+From [Wikipedia](https://en.wikipedia.org/wiki/Domain_drop_catching):
+
+> Domain drop catching, also known as domain sniping, is the practice of registering a domain name once registration has lapsed, immediately after expiration.
+
+## Why do you not allow drop catching?
+
+Our upstream domain registration provider limits the number of checks we are allowed to execute. If you attempt to engage in drop catching then this will impact all of our customers. 
```

Comment:
Instead of just "number of checks" should we say "number of registration checks"? I don't know if "check" is a term that's commonly used in this scenario, so feel free to disregard if it is.